components/aws/sagemaker/tests/integration_tests/environment.yml

```diff
@@ -7,8 +7,7 @@ dependencies:
   - pip=20.0.*
   - awscli=1.18.*
   - boto3=1.12.*
-  - pytest=5.*
-  - pytest-xdist=1.31.*
+  - pytest-xdist=2.1.0
```

Comment:
removed pytest ?

Reply:
`pytest-xdist` listed it as a dependency. I thought I'd leave it to that to add its version.